addons/website/static/src/js/utils.js

```diff
@@ -351,6 +351,32 @@ async function svgToPNG(src) {
     }).then(loadedImgEl => toPNGViaCanvas(loadedImgEl));
 }
 
+function generateGMapIframe() {
+    const iframeEl = document.createElement('iframe');
+    iframeEl.classList.add('s_map_embedded', 'o_not_editable');
+    iframeEl.setAttribute('width', '100%');
+    iframeEl.setAttribute('height', '100%');
+    iframeEl.setAttribute('frameborder', '0');
+    iframeEl.setAttribute('scrolling', 'no');
+    iframeEl.setAttribute('marginheight', '0');
+    iframeEl.setAttribute('marginwidth', '0');
+    iframeEl.setAttribute('src', 'about:blank');
```

Comment:
Actually, I don't mind if it stays, but does this have any advantage over not adding an src?

Reply:
Didn't really checked, I just kept the code as it was.
But actually it needs to remain, I just had a quick check and no src iframe are not valid.
https://stackoverflow.com/a/5946631